cirras-underwriting-api/cirras-underwriting-model/src/main/java/ca/bc/gov/mal/cirras/underwriting/model/v1/VerifiedYieldContract.java

```diff
@@ -0,0 +1,38 @@
+package ca.bc.gov.mal.cirras.underwriting.model.v1;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+public interface VerifiedYieldContract<A extends AnnualField> extends Serializable {
```

Comment:
Is _extends AnnualField_ necessary?

Reply:
I wasn't sure. It depends on the questions we asked in PIM-1699. We can either included a "fields" member like we do for InventoryContract and DopContract, in which case we do need this, or use an endpoint to load the fields for the contract separately, in which case we probably don't need this.

Reply:
Ok, let's leave it in until we have all the answers we need.